src/Geno/C++/GUI/Modals/IModal.h

```diff
@@ -20,6 +20,8 @@
 
 #include <Common/Macros.h>
 
+#include "Auxiliary/ImGuiAux.h"
+
```

Comment:
Internal includes before external includes

Reply:
will fix it